media/static-bundles.json

```diff
@@ -842,13 +847,16 @@
     },
     {
       "files": [
-        "js/firefox/whatsnew/up-to-date.js",
-        "js/libs/spin.min.js",
-        "js/base/send-to-device.js",
-        "js/firefox/whatsnew/whatsnew-82.js"
+        "js/firefox/whatsnew/up-to-date.js"
```

Comment:
I don't think you meant to edit the 82 JS file?

Reply:
I did – per @craigcook's [comment](https://github.com/mozilla/bedrock/pull/9588#discussion_r514487587) that 82 also wasn't using the `send-to-device`/`spin` files.